rust_src/build.rs

```diff
@@ -31,14 +32,11 @@ fn get_function_name(line: &str) -> Option<String> {
         }
     }
 
-    None
+    panic!("Failed to find function name in: '{}'", line)
```

Comment:
function can now return `String` directly?

Reply:
Good point. Although I meant move the panic into the main line.

Reply:
Bah phones. Might not meant.